### DISCUSSION TITLE
Apache Thrift Support?

### DISCUSSION THREAD

**Discussion by 0zl** [mitmproxy/mitmproxy#5822] [Q&A]
Hi,

I'm currently reverse engineering one application that uses Thrift as their response from a TCP connection. Is there any support or plug-in to be able to read data from the Thrift protocol?

Thank you!

**Comment by rosydawn6** [comment]
Perhaps you could integrate this into your mitmproxy addon: https://thrift.apache.org/tutorial/py.html

**Comment by mhils** [comment]
> Is there any support or plug-in to be able to read data from the Thrift protocol?

mitmproxy supports raw TCP, but nothing Thrift-specific beyond that at the moment.
